### PR TITLE
added "Play Sound" switch to the context menu

### PR DESCRIPTION
**Pull request by jfwiebe**
I added an extra menu checkbox for enabling and disabling the beep sound.

image

<sub>PS: This is my first open-source commit, I am really excited.</sub>😄

### CODE REVIEW

File: pretixdroid/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <resources>
+    <string name="action_play_sound">Play Sounds</string>
```
Comment (raphaelm) [inline pretixdroid/app/src/main/res/values/strings.xml]: Sounds -> sounds

File: pretixdroid/app/src/main/java/eu/pretix/pretixdroid/AppConfig.java [hunk 2]
```diff
@@ -49,11 +50,19 @@ public boolean getAutofocus() {
         return prefs.getBoolean(PREFS_KEY_AUTOFOCUS, true);
     }
 
+    public boolean getSound() {
```
Comment (raphaelm) [inline pretixdroid/app/src/main/java/eu/pretix/pretixdroid/AppConfig.java]: I think "getSoundEnabled" would be a better name, what do you think? (Same for setSoundEnabled)

**Review by raphaelm — COMMENTED** [review]
Nice! This looks very good! I left two comments in there which are purely cosmetic. After that, we're good to go here :)

**Comment by jfwiebe** [comment]
Thanks, I totally agree with you, changes have been implemented.

**Comment by raphaelm** [comment]
\o/ Thank you very much and congratulations to your first PR!

**Pull request merged**